resources/web/template.html

```diff
@@ -82,22 +82,48 @@
         <div class="content-wrapper">
 
           <section id="guide" <!-- DOCS LANG -->>
-            <div class="container">
-              <div class="row">
-                <div class="col-xs-12 col-sm-8 col-md-8 guide-section">
+            <div class="container-fluid">
+              <div class="row pb-3">
+                <div class="col-12 order-2 col-md-4 order-md-1 col-lg-3 h-almost-full-md sticky-top-md" id="left_col">
+                  <!-- The TOC is appended here -->
+                </div>
+
+                <div class="col-12 order-1 col-md-8 order-md-2 col-lg-7 order-lg-2 guide-section" id="middle_col">
                   <!-- start body -->
                   <!-- DOCS BODY -->
                   <!-- end body -->
                 </div>
-                <div class="col-xs-12 col-sm-4 col-md-4" id="right_col">
-                  <div id="rtpcontainer" style="display: block;">
-                    <div class="mktg-promo">
-                      <h3>Most Popular</h3>
-                      <ul class="icons">
-                        <li class="icon-elasticsearch-white"><a href="https://www.elastic.co/webinars/getting-started-elasticsearch?baymax=default&elektra=docs&storm=top-video">Get Started with Elasticsearch: Video</a></li>
-                        <li class="icon-kibana-white"><a href="https://www.elastic.co/webinars/getting-started-kibana?baymax=default&elektra=docs&storm=top-video">Intro to Kibana: Video</a></li>
-                        <li class="icon-logstash-white"><a href="https://www.elastic.co/webinars/introduction-elk-stack?baymax=default&elektra=docs&storm=top-video">ELK for Logs & Metrics: Video</a></li>
-                      </ul>
+
+                <div class="col-12 order-3 col-lg-2 order-lg-3 h-almost-full-lg sticky-top-lg" id="right_col">
+                  <div id="sticky_content">
+                    <!-- The OTP is appended here -->
+                    <div class="row">
+                      <div class="col-0 col-md-4 col-lg-0" id="bottom_left_col"></div>
+                      <div class="col-12 col-md-8 col-lg-12">
+                        <div id="rtpcontainer">
```

Comment:
These changes will only affect the style for people who don't load content from Optimizely (I think?) I'd be curious to see how actual changes from Optimizely or the recommender look.

Reply:
That's correct. We still need to work with marketing to figure out how to test the Optimizely changes. Staging sounds like it could be our best bet.

Reply:
A very detailed issue coming soon in /website-requests! 💁‍♀️

Reply:
✨  https://github.com/elastic/website-requests/issues/678

Reply:
> I don't feel particularly qualified to review the CSS or JavaScript changes

I'd love to find a front-end engineer to review this. At a bare minimum, I'm confident there's a better way to scroll the TOC than what I implemented in 936464bcffb42bb4f17809dc919f95665ba4a0b2.

Reply:
That scrolling logic looks good to me! jQuery used to have a way to wait for an element selector to resolve, but doesn't look like that's still the case.

Reply:
Ah, great! Thank you so much for taking a look.